### DISCUSSION TITLE
Copy to clipboard

### DISCUSSION THREAD

**Discussion by mattmattkim** [flyerhq/flutter_chat_ui#462] [Ideas & Requests] [answered]
Hi team,

Could't find anything about copy to clipboard on particular messages. 
I was wondering if this is planned or some guidance on the best place to customize this on my own?

Thank you!

**Comment by demchenkoalex** [comment]
Hello.

You mean copy text?

**Reply by mattmattkim** [reply]
yes, a button to copy the text of a message.

**Reply by demchenkoalex** [reply]
I didn't see any buttons to copy content of the message in popular messengers (aside from the button on the menu when you long press a message, but menu is not part of the chat currently). So I used another pattern - make text on the message selectable and then you can use system popup to copy text, when you select. This should be enabled by default. There is `isTextSelectable` property in the text message options, which is true by default.

**Comment by demchenkoalex** [comment, marked as answer]
pointing to this https://github.com/flyerhq/flutter_chat_ui/discussions/93
